docs/user-guide/container-environment.md

```diff
@@ -68,7 +68,7 @@ There are two types of information that are available within the container envir
 
 ### Container Information
 
-Currently, the only information about the container that is available to the container is the Pod name for the pod in which the container is running.  This ID is set as the hostname of the container, and is accessible through all calls to access the hostname within the container (e.g. the hostname command, or the [gethostname][1] function call in libc).  Additionally, user-defined environment variables from the pod definition, are also available to the container, as are any environment variables specified statically in the Docker image.
+Currently, the Pod name for the pod in which the container is running is set as the hostname of the container, and is accessible through all calls to access the hostname within the container (e.g. the hostname command, or the [gethostname][1] function call in libc).  The Pod name and namespace are also available as environment variables via the [downward API](downward-api.md).  Additionally, user-defined environment variables from the pod definition, are also available to the container, as are any environment variables specified statically in the Docker image.
```

Comment:
Thanks.

Actually, as long as we're here, could we please add a note that users should not rely on the fact that the hostname is set to the pod name, since we plan to change that?